src/templates/app/user/settings/professional.jsx

```diff
@@ -19,7 +19,8 @@ const Professional = () => (
                 <p>{it.L('You are categorised as a professional client.')}</p>
             </div>
             <div className='invisible' id='rejected'>
-                <p>{it.L('Your request to be treated as a professional client has been rejected. Please check your inbox for more details. Your account remains categorised as a retail client account.')}</p>
+                <p>{it.L('Your request to be treated as a  professional is [_1]not approved[_2].', '<strong>', '</strong>')}</p>
```

Comment:
there are two spaces before `professional`, and are we missing the word `client` after that?

Reply:
as per the design only `approved` is supposed to be bold?

Reply:
but I agree that making `not approved` bold is better, perhaps you can check with Lucas

Reply:
Already checked with lucas earlier. We should be bolding `not approved`.

Reply:
@negarn